### ISSUE TITLE
web version timestamp truncated

### ISSUE THREAD

**Issue by sj0000e** [daegalus/dart-uuid#106]
Dart JS supports only 32-bit bit manipulation. Therefore, UUID v7 does not work correctly.

**Comment by daegalus** [comment]
I'll need more info, as my manipulations stick to 32bit and below. What's the actual error or problem?

**Comment by sj0000e** [comment]
uuid: ^4.3.2
```
void main() {
  print(const Uuid().v7());
  print(const Uuid().v7());
}
```

I wanted to use UUID v7, generate them on multiple machines, and sort them chronologically, 
javascript version generated an invalid ke, so I couldn't sort chronologically.
```
<javascript>
f3b80000-2d78-7e3e-9214-d7b3935cab87
f3b80000-2d79-7fee-9238-9c2a18c4d2f1

<macOs>
flutter: 018cf3b7-9411-7ebf-bdc2-bf9e957ac16f
flutter: 018cf3b7-942d-7673-88ef-5abe52ba7c14
```

**Comment by daegalus** [comment]
Ok, I can confirm it's truncation. I fixed it for v7 but found a related bug in v6. But it's trickier to fix, but I should have a new version with fixes soon

**Comment by daegalus** [comment]
fixed in 4.3.3
